h/schemas/forms/admin/group.py

```diff
@@ -119,7 +126,7 @@ def group_organization_select_widget(node, kw):
 class CreateAdminGroupSchema(CSRFSchema):
```

Comment:
At this point, are both the create and edit forms using this schema? Is a rename or comment in order? Or is it obvious enough?

Reply:
This is a pretty fair point. it's probably just the `AdminGroupSchema` if it's used equally for both.

Reply:
This has been renamed to just "AdminGroupSchema" now